### PR TITLE
feature: ux changes

### PR DESCRIPTION
**Pull request by martinpastore**
Some UX "improvements"

* Added space on profile info.
* Change 'register' by 'my profile' when the user is logged in
* Avoid rendering pagination when there is only one page. 

Captura de Pantalla 2020-10-24 a la(s) 10 24 46
Captura de Pantalla 2020-10-24 a la(s) 10 25 17
Captura de Pantalla 2020-10-24 a la(s) 10 27 35

### CODE REVIEW

File: src/components/commons/Footer/Footer.jsx [hunk 1]
```diff
@@ -20,9 +24,14 @@ const Footer = () => {
...
-              <Button handleClick={handleGoToRegister} bigButton text={t('common:signUp')} />
-            </div>
+            {authStore.isLogin ?
+              <div className={styles.containerButtonSignUp}>
+                <Button handleClick={handleGoToProfile} bigButton text={t('navbar:myProfile')} />
+              </div>
+              :
+              <div className={styles.containerButtonSignUp}>
+                <Button handleClick={handleGoToRegister} bigButton text={t('common:signUp')} />
+              </div>}
```
Comment (alexrobaina) [inline src/components/commons/Footer/Footer.jsx:34]: do you think it's ok if change for this?
````
          <div className={styles.containerButtonSignUp}>
              <Button
                bigButton
                text={authStore.isLogin ? t('navbar:myProfile') : t('common:signUp')}
                handleClick={authStore.isLogin ? handleGoToProfile : handleGoToRegister}
              />
            </div>
````

**Review by alexrobaina — COMMENTED** [review]
Thanks you very much for these changes, great get better!
If it's okay with you, let's merge after these change 😃

File: src/components/commons/Footer/Footer.jsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+import UserContext from 'Context/UserContext'
 
 const Footer = () => {
   const history = useHistory()
   const { t } = useTranslation('landingPage')
+  const { authStore } = useContext(UserContext);
+
+  const handleGoToProfile = useCallback(() => history.push(`${PROFILE_USER}/${authStore.user._id}`), [])
 
   const handleGoToRegister = useCallback(() => {
```
Comment (alexrobaina) [inline src/components/commons/Footer/Footer.jsx:17]: It's okay for you to change this functions for these? 😃
````
  const goTo = useCallback(() => {
    if (authStore.isLogin) return history.push(`${PROFILE_USER}/${authStore.user._id}`)
    history.push(REGISTER)
  }, [authStore.isLogin])

  const formatText = useCallback(() => {
    if (authStore.isLogin) return t('navbar:myProfile')
    t('common:signUp')
  }, [authStore.isLogin])
````

File: src/components/commons/Footer/Footer.jsx (hunk 1 above)
Comment (alexrobaina) [inline src/components/commons/Footer/Footer.jsx:34]: The button would look like this: 
````
<div className={styles.containerButtonSignUp}>
              <Button handleClick={goTo} bigButton text={formatText()} />
            </div>
````

File: src/containers/LandingPage/FirstSection/FirstSection.jsx [hunk 3]
```diff
@@ -17,11 +18,14 @@ const FirstSection = () => {
   const animationImage = useAnimation()
   const animationButtonDown = useAnimation()
   const [ref] = useInView({ threshold: 0.1 })
+  const { authStore } = useContext(UserContext);
 
   const gotToRegister = useCallback(() => {
     history.push(REGISTER)
   }, [])
 
+  const goToProfile = useCallback(() => history.push(`${PROFILE_USER}/${authStore.user._id}`), [])
```
Comment (alexrobaina) [inline src/containers/LandingPage/FirstSection/FirstSection.jsx:27]: equal to footer changes:
````
  const goTo = useCallback(() => {
    if (authStore.isLogin) return history.push(`${PROFILE_USER}/${authStore.user._id}`)
    history.push(REGISTER)
  }, [authStore.isLogin])

  const formatText = useCallback(() => {
    if (authStore.isLogin) return t('navbar:myProfile')
    t('common:signUp')
  }, [authStore.isLogin])
````

**Review by alexrobaina — COMMENTED** [review]
Hi @martinpastore, I suggested some changes, if it's okay with you we can merge after those changes.

Thanks for these great collaboration

File: src/components/commons/CardGoogle/CardGoogle.jsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types'
 import { HOST, AWS_STORAGE } from 'services/config'
 import noImage from './noimg.png'
 import styles from './cardGoogle.scss'
+import { PROFILE_USER } from 'routing/routes'
```
[the remark this answers is not in the document]
Comment (martinpastore) [reply src/components/commons/CardGoogle/CardGoogle.jsx]: Didn't see it. fixed now :D

File: src/components/commons/Footer/Footer.jsx [hunk 5]
```diff
@@ -1,14 +1,18 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useContext } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useHistory } from 'react-router'
-import { REGISTER } from 'routing/routes'
+import { PROFILE_USER, REGISTER } from 'routing/routes'
 import SocialMediaButtons from 'components/commons/SocialMediaButtons'
 import Button from 'components/commons/Button'
 import styles from './footer.scss'
+import UserContext from 'Context/UserContext'
```
[the remark this answers is not in the document]
Comment (martinpastore) [reply src/components/commons/Footer/Footer.jsx:7]: Same! :D

File: src/containers/ProfilePets/ProfilePets.jsx [hunk 6]
```diff
@@ -15,6 +15,7 @@ import TabViewInformationPet from 'components/commons/TabViewInformationPet'
 import Footer from 'components/commons/Footer/index'
 import noImage from './noImage.svg'
 import styles from './profilePets.scss'
+import { PROFILE_USER } from 'routing/routes'
```
[the remark this answers is not in the document]
Comment (martinpastore) [reply src/containers/ProfilePets/ProfilePets.jsx]: Same :D

File: src/components/commons/Footer/Footer.jsx (hunk 2 above)
Comment (martinpastore) [reply src/components/commons/Footer/Footer.jsx]: looks cleaner, I've changed it

File: src/components/commons/Footer/Footer.jsx (hunk 1 above)
Comment (martinpastore) [reply src/components/commons/Footer/Footer.jsx]: agreed.

File: src/containers/LandingPage/FirstSection/FirstSection.jsx (hunk 3 above)
Comment (martinpastore) [reply src/containers/LandingPage/FirstSection/FirstSection.jsx]: same here! I've addressed this change

**Review by alexrobaina — COMMENTED** [review]
Great! I'm going to merge. 
Thank you!

**Pull request merged**